plugins/convention/src/test/java/org/apache/struts2/convention/PackageBasedActionConfigBuilderTest.java

```diff
@@ -921,7 +921,11 @@ public void inject(Object o) {
         }
 
         public <T> T inject(Class<T> implementation) {
-            return null;
+            try {
+                return implementation.newInstance();
```

Comment:
Because with use this fake implementation in tests and now `ObjectFactory` will use `Container#inject()` to instantiate a new object. Instead I've got `nulll`s.